web-no-modal-sdk/quick-starts/nuxt-no-modal-quick-start/components/Home.vue

```diff
@@ -0,0 +1,281 @@
+<template>
+  <div id="app">
+    <h2>
+      <a target="_blank" href="https://web3auth.io/docs/sdk/pnp/web/modal" rel="noreferrer"> Web3Auth </a>
```

Comment:
All looks good. Just change everything(sdk link, example link etc.) to no-modal in the no-modal example.

Reply:
updated